.github/workflows/crowdin-upload.yml

```
@@ -0,0 +1,64 @@
+      - master
+      - main
+  pull_request:
+    branches:
+      - master
+      - main
+  workflow_dispatch:  # Allow manual triggering
+
+jobs:
+  extract-and-upload:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)